index.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Fedora API Specification</title>
+    <title>Fedora API Specification 1.0.0</title>
     <meta charset='utf-8'>
-    <meta name="description" content="Fedora API Specification">
-    <script src='respec-w3c-common.js'
```

Comment:
Should we remove the old `respec-w3c-common.js` from this GitHub repo? It would be good to document which version of ReSpec we are based on... noting that the more recent releases of ReSpec introduce breaking changes for us.

Reply:
yes, have removed